### PR TITLE
Modify, Initializer

### PR DESCRIPTION
**Pull request by Pluu**
## Issue
- N/A

## Overview (Required)
- More and more code is added to the application class. I'm suggest ways to improve this.
- You can wire initialization code using `Dagger IntoSet`.

## Links
- [chrisbanes/tivi ~ AppInitializers](https://github.com/chrisbanes/tivi/blob/master/app/src/main/java/app/tivi/appinitializers/AppInitializers.kt)

### CODE REVIEW

File: android-base/src/main/java/io/github/droidkaigi/confsched2020/di/AppModuleBinds.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    @IntoSet
+    abstract fun provideCoilInitializer(bind: CoilInitializer): AppInitializer
+
+    @Binds
+    @IntoSet
+    abstract fun provideEmojiInitializer(bind: EmojiInitializer): AppInitializer
+
+    @Binds
+    @IntoSet
+    abstract fun provideFirebaseFirestoreInitializer(bind: FirebaseFirestoreInitializer): AppInitializer
```
[the remark this answers is not in the document]
Comment (Pluu) [reply android-base/src/main/java/io/github/droidkaigi/confsched2020/di/AppModuleBinds.kt]: Fixed, rename

**Comment by tetsukay** [comment]
👀

**Comment by tetsukay** [comment]
LGTM!

File: android-base/src/main/java/io/github/droidkaigi/confsched2020/initializer/AppInitializers.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package io.github.droidkaigi.confsched2020.initializer
+
+import android.app.Application
+import javax.inject.Inject
+
+class AppInitializers @Inject constructor(
+    private val initializers: Set<@JvmSuppressWildcards AppInitializer>
```
Comment (takahirom) [inline android-base/src/main/java/io/github/droidkaigi/confsched2020/initializer/AppInitializers.kt]: Is `@JvmSuppressWildcards` still needed? 👀 
https://youtu.be/o-ins1nvbDg?t=655

Comment (Pluu) [reply android-base/src/main/java/io/github/droidkaigi/confsched2020/initializer/AppInitializers.kt]: @takahirom I understand that the above feature is not supported.
```
e: /.../android-base/build/tmp/kapt3/stubs/debug/io/github/droidkaigi/confsched2020/di/AppComponent.java:8: error: [Dagger/MissingBinding] java.util.Set<? extends io.github.droidkaigi.confsched2020.initializer.AppInitializer> cannot be provided without an @Provides-annotated method.
public abstract interface AppComponent extends dagger.android.AndroidInjector<io.github.droidkaigi.confsched2020.App>, io.github.droidkaigi.confsched2020.di.AppComponentInterface {
                ^
      java.util.Set<? extends io.github.droidkaigi.confsched2020.initializer.AppInitializer> is injected at
          io.github.droidkaigi.confsched2020.initializer.AppInitializers(initializers)
      io.github.droidkaigi.confsched2020.initializer.AppInitializers is injected at
          io.github.droidkaigi.confsched2020.App.initializers
      io.github.droidkaigi.confsched2020.App is injected at
          io.github.droidkaigi.confsched2020.di.AppComponent.inject(io.github.droidkaigi.confsched2020.App)
```

**Comment by takahirom** [comment]
LGTM!!

**Pull request merged**